test/gtest/ucp/test_ucp_am.cc

```diff
@@ -1848,7 +1849,9 @@ class test_ucp_am_nbx_rndv_memtype : public test_ucp_am_nbx_rndv {
 
 UCS_TEST_P(test_ucp_am_nbx_rndv_memtype, rndv)
 {
-    test_am_send_recv_memtype(64 * UCS_KBYTE);
+    ucp_context_config_t *cfg = &sender().worker()->context->config.ext;
+    size_t size               = cfg->rndv_frag_size[UCS_MEMORY_TYPE_HOST] + 1;
+    test_am_send_recv_memtype(size);
```

Comment:
maybe worth testing several sizes that are "interesting"?

Reply:
can you sugget some?
this test has many variants, i'd try not to add a lot to keep CXI time reasonable

Reply:
> can you sugget some?

at least do not remove old values, usually init/cleanup takes significat part of time in comparison with send/recv

Reply:
do you mean to also test 64kb as before?

Reply:
yes: 64k, frag_size+1, frage_size-1

Reply:
updated + squashed, because the PR is small